webview-ui/src/manualTest/clusterPropertiesTests.tsx

```diff
@@ -115,6 +116,21 @@ export function getClusterPropertiesScenarios() {
             });
             webview.postGetPropertiesResponse(testClusterInfo);
         }
+
+        async function handleAbortAgentPoolOperation(agentPoolName: string, hasError: boolean) {
+            await new Promise(resolve => setTimeout(resolve, 1000));
+            if (hasError) {
+                webview.postErrorNotification("Sorry, I wasn't quite able to abort the operation.");
```

Comment:
Can we finetune this error msg ? ex: ```Abort agent pool operation failed due to error``` ?

Reply:
Sure, whatever here is fine **because** this is **test** error so any error here is just for testing sake no end to end functionality is in direct impact. Do we need consensus - if no replies I could just change it to whatever recommended. thanks.

Reply:
Sorry didn't realize this is test file. I will resolve this comment.